container/network.go

```diff
@@ -33,36 +29,29 @@ func CreateNetwork(namespace []string) (networkID string, err error) {
 		},
 	})
 	if err != nil {
-		return
+		return "", err
 	}
-	networkID = response.ID
-	return
+	return response.ID, nil
 }
 
-// DeleteNetwork deletes a Docker Network associated with a namespace
-func DeleteNetwork(namespace []string) (err error) {
+// DeleteNetwork deletes a Docker Network associated with a namespace.
+func DeleteNetwork(namespace []string) error {
 	network, err := FindNetwork(namespace)
-	if docker.IsErrNotFound(err) {
-		err = nil
-		return
-	}
-	if err != nil {
-		return
+	if err != nil && !docker.IsErrNotFound(err) {
```

Comment:
this is not the same behavior than before
before: if the network is not found we exit with no error
now: if the network is not found we continue and probably have a crash on the `network.ID`

Reply:
No, the behavior is the same as before:
The tests pass on this function

Reply:
Fixed 👍